Fixing broken hyperlinks and typos in Basics section (#184)
neuvector/docs: docs/01.basics/02.requirements/02.requirements.md

* Docker run-time version: 1.9.0 and up; Docker API version: 1.21, CE and EE.
* Containerd and CRI-O run-times (requires changes to volume paths in sample yamls). For more information, refer to [Kubernetes deployment](../../02.deploying/02.kubernetes/02.kubernetes.md) and [OpenShift deployment](../../02.deploying/04.openshift/04.openshift.md).
* NeuVector is compatible with most commercially supported CNI's. Officially tested and supported are openshift ovs (subnet/multitenant), calico, flannel, cilium, antrea and public clouds (gke, aks, iks, eks). Support for Multus was added in v5.4.0.
* Console: Chrome or Firefox browser recommended. IE 11 not supported due to performance issues.
* Minikube is supported for simple initial evaluation but not for full proof of concept. For more information, refer to [changes required for the Allinone yaml to run on Minikube](#minikube).

> AWS Bottlerocket Note: Must change path of the containerd socket specific to Bottleneck. Please refer to [Kubernetes deployment](../../02.deploying/02.kubernetes/02.kubernetes.md)

Reviewer: ```suggestion
:::note
AWS Bottlerocket: You must change the path of the containerd socket specific to Bottlerocket. Please refer to the [Kubernetes deployment](../../02.deploying/02.kubernetes/02.kubernetes.md) documentation for more information.
:::
```

For throughput or latency sensitive environments, additional memory and/or a dedicated CPU core can be allocated to the NeuVector Enforcer container.

> For performance tuning of the Controller and Scanner for registry scanning, refer the System Requirements above.

Reviewer: ```suggestion
For performance tuning of the Controller and Scanner for registry scanning, refer to the [System Requirements](#system-requirements) above.
```

> For additional advice on performance and sizing, refer the [Onboarding/Best Practices section](/deploying/production?target=_blank#best-practices-tips-qa-for-deploying-and-managing-neuvector).

Reviewer: ```suggestion
For additional advice on performance and sizing, refer to the [Onboarding/Best Practices section](/deploying/production?target=_blank#best-practices-tips-qa-for-deploying-and-managing-neuvector).
```
